n2v/grid/grider.py

```diff
@@ -19,22 +19,24 @@
 
 from .cubeprop import Cubeprop
 
-default_grid = np.concatenate((np.linspace(-10,10,20, dtype=np.half)[:,None],
+_default_grid = np.concatenate((np.linspace(-10,10,20, dtype=np.half)[:,None],
```

Comment:
I initially had intended for "default_grid" to be the default argument of each function for a "quick grid plot." But if there won't be any default argument, then there is no point in keeping this array. I am fine with either option.